css/style.css

```diff
@@ -0,0 +1,144 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+}
+
+/* General class */
+.flex {
+  display: flex;
+}
+
+/* This is the header styles */
+header {
+  background-image: url(../assets/images/expedition_hero.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 80vh;
+  padding-top: 40px;
+}
```

Comment:
- [ ] **OPTIONAL** You can also use the `background-position` property to fix your image in a certain position, for example, `center`. [More here](https://developer.mozilla.org/es/docs/Web/CSS/background-position)